### ISSUE TITLE
NotoMono has only Regular weight

### ISSUE THREAD

**Issue by jriddell** [googlei18n/noto-fonts#716]
NotoMono has only a regular face on it, to complete this font it should have bold and italic variants.

This is used by applications for highlighting, e.g. Kile https://phabricator.kde.org/D1816

**Comment by dougfelt** [comment]
Mono in phase 3 is planned to have multiple weights, and will likely add a condensed version.  No italic is scheduled.  We don't yet have alpha drops of the new Mono.

BTW, the character repertoire for Mono is not the same as LGC, though there is significant overlap.  You can see the currently planned repertoire by examining the data in the nototools/nototools/noto_cmap_phase3.xml file.  We hope to extend this to include some Math characters after work on that starts but this is still tentative, see issue #669.

**Comment by XavierCLL** [comment]
+1 multiple weights in NotoMono is very neccesary for IDEs for example. Currently I use NotoMono and I have noticed the failure of multiple weights.

**Comment by rastus-vernon** [comment]
I have seen editors and websites use italics for comments in source code.

**Comment by dougfelt** [comment]
@xiangye, please see above rationale for providing an italic version of Noto Monospace.

**Comment by xiangyexiao** [comment]
Mono Italic is something we can consider after we finish the current work on Phase 3 T1 and T2 languages.

**Comment by marekjez86** [comment]
multiple weights are in https://github.com/googlei18n/noto-fonts/tree/master/alpha/from-pipeline/unhinted/ttf/sans

**Comment by kphillisjr** [comment]
I agree that the Noto Sans Mono font (Noto Sans Mono) needs to have more variations. I am excited to see that the bold (and similar) variations are in testing ( see the Alpha fonts ). The only thing I noticed is that there is no Italic variation, and for the most part adding this would bring the typeface feature parity to the same level as Noto Sans and Noto Serif.

Also, The previous link leads to a 404 page due to having the Alpha fonts moved. The updated link to where these files can be found at: https://github.com/googlei18n/noto-fonts-alpha
- TrueType File ( TTF): https://github.com/googlei18n/noto-fonts-alpha/tree/master/from-pipeline/unhinted/ttf/sans/NotoSansMono
- OpenType File ( OTF) : https://github.com/googlei18n/noto-fonts-alpha/tree/master/from-pipeline/unhinted/otf/sans/NotoSansMono
